frontend/src/store/account.js

```diff
@@ -115,6 +116,8 @@ const mutations = {
         state.user = user
     },
     setTeam (state, team) {
```

Comment:
Is it possible that setTeam is being called with `null`? Suggest wrapping in try/catch to rule out.